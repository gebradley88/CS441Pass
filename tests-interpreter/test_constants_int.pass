// declares an integer constant and then produces an error when attempting to change it

const int n = 31415;
print x;
x = 2012;
print x;
