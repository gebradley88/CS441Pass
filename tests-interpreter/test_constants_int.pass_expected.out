Variable 'x' undeclared at line 4
